hw02/PlayerController.cs

```diff
@@ -18,34 +22,65 @@ 4. Jump Speed as a float
     
     When the game starts, we would like to find our character. In our Unity Editor, we have a 3D model of our player and the player is 
     represented as a component in Unity. How do we grab our player controller in code and where should we write this line of code in this
-    C# document (PlayerController.cs)?
+    C# document (PlayerController.cs)? 
     */
-    
+
+    public string characterName;
+    public float speed;
+    public float gravityValue;
+    public float jumpSpeed;
+    public PlayerController myPlayerController;
+
     // Start is called before the first frame update
-    void Start()
+    private void Start()
     {
+        // Answer: myPlayerController = GameObject.Find("Player").GetComponent<PlayerController>();
```

Comment:
That's correct! Using gameobject.find() will search every existing objects in the scene and return the object with the name "Player".